packages/sdk/src/sdk/api/users/UsersApi.ts

```diff
@@ -54,6 +58,98 @@ export class UsersApi extends GeneratedUsersApi {
     this.logger = logger.createPrefixedLogger('[users-api]')
   }
 
+  /** @hidden
+   * Generate a new user id for use in creation flow
+   */
+  async _generateUserId() {
```

Comment:
Is the underscore to declare it as internal? Maybe we can do `private` then

I added a similar generateTrackId in the upload pr, left that one as public because we call it manually in the upload sagas

Reply:
Oh yes, that was the intention. I forgot that private semantics exist in JS/TS classes.